src/primer_filter.cpp

```diff
@@ -17,100 +17,94 @@ PrimerFinder::~PrimerFinder() {
     // nothing to do
 }
 
-const vector<PrimerPair>& PrimerFinder::get_primer_pairs() const {
-    return primer_pairs;
+const vector<PrimerPair>& PrimerFinder::get_primer_pairs_of_chrom(const string chrom_name) const {
```

Comment:
Not the hugest deal here, but remember to use references when passing strings around